packages/ListBoxWithTags/src/useWithTags.js

```diff
@@ -102,15 +102,17 @@ export default function useWithTags({
   }
 
   return {
+    listBoxWithTagsProps: {
```

Comment:
nit: just wonder if this may be better being called `spreadableMoreProps` or `validMoreProps`? 🤔

Reply:
I thought with the name `listBoxWithTags`  it's easier to understand where we should put these props in